Common/SAM.h

```diff
@@ -339,10 +339,11 @@ struct SAMRecord : SAMAlignment {
 			>> o.cigar >> o.mrnm >> o.mpos >> o.isize;
 #if SAM_SEQ_QUAL
 		in >> o.seq >> o.qual;
-		if (in.peek() != '\n')
-			in >> o.tags;
-#endif
+		getline(in, o.tags);
+		o.tags = o.tags.substr(1);
```

Comment:
```suggestion
		getline(in >> Skip('\t'), o.tags);
```
`substr` makes a copy of the string which is requires a heap allocation and is unnecessarily inefficient. `o.tags.erase(0, 1);` would be better, and the above suggestion is better still.

Reply:
``
```suggestion
                in.ignore(1);
		getline(in, o.tags);
```
What about this change? Or is the skip version still preferred?